.setup/ansible/roles/submitty_install/files/append_to_submitty.conf

```diff
@@ -0,0 +1,23 @@
+<VirtualHost *:80>
+    ServerName __your_domain__
+    Redirect / https://__your_domain__/
```

Comment:
I'm not familiar with Ansible, but I'm familiar with other similar products where it's possible to inject variables into files.  Is it possible to automatically inject the domain from a central config location here?

Reply:
Yes, that happens in the submitty_install.yml file on lines 92-98, and takes the values from either the defaults file, or the submitty_install playbook.